ietf-microwave-radio-link.yang

```diff
@@ -315,9 +393,13 @@ module ietf-microwave-radio-link {
       description
         "Shows the operative status of the transmitter.";
     }
-
-    uses mw-types:common-microwave-properties;
-
+    leaf tx-frequency {
+      type uint32;
+      units "kHz";
+      mandatory true;
+      description
+        "Selected transmitter frequency.";
+    }
```

Comment:
Why the common-microwave-properties grouping is no longer used?

I am wondering whether some of the previous changes to the main branch got lost